### PR TITLE
Seed the random number generator on AWS 

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: upup/pkg/fi/nodeup/command.go [hunk 1]
```diff
@@ -674,16 +680,60 @@ func loadKernelModules(context *model.NodeupModelContext) error {
 	return nil
 }
 
-// getNodeConfigFromServer queries kops-controller for our node's configuration.
-func getNodeConfigFromServer(ctx context.Context, config *nodeup.ConfigServerOptions) (*nodeup.BootstrapResponse, error) {
-	var authenticator fi.Authenticator
-
+// getRegionAndSeedRNG queries the cloud provider for the region and adds entropy to the random number generator.
```
Comment (hakman) [inline upup/pkg/fi/nodeup/command.go:683]:
```suggestion
// getRegion queries the cloud provider for the region.
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: pkg/model/tests/data/bootstrapscript_0.txt [hunk 2]
```diff
@@ -172,6 +172,7 @@ __EOF_CLUSTER_SPEC
 
 cat > conf/kube_env.yaml << '__EOF_KUBE_ENV'
 AuxConfigHash: /0fgYG0lsuxVGuNdtGBZA0RYxOfvLD/xOFg1eJwyfcw=
+CloudProvider: ""
```
[the remark this answers is not in the document]
Comment (johngmyers) [reply pkg/model/tests/data/bootstrapscript_0.txt:175]: Possibly. It's outside the scope of this PR, though.

**Review by hakman — APPROVED** [review]

File: pkg/model/tests/data/bootstrapscript_0.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (johngmyers) [reply pkg/model/tests/data/bootstrapscript_0.txt:175]: The cloud provider is set in code which is outside the scope of these tests.

**Pull request merged**